src/storybook/patterns/dropdown-in-modals/dropdown-in-modals.stories.mdx

```diff
@@ -0,0 +1,247 @@
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import { useCallback, useState, useMemo, useRef } from "react";
+import { ComponentNameDecorator } from "../../components";
+import { OpenModalButton } from "./helpers";
+import ModalContent from "../../../components/Modal/ModalContent/ModalContent";
+import Dropdown from "../../../components/Dropdown/Dropdown";
+import Modal from "../../../components/Modal/Modal";
+import Flex from "../../../components/Flex/Flex";
+import Dialog from "../../../components/Dialog/Dialog";
+import Button from "../../../components/Button/Button";
+import { Link } from "vibe-storybook-components";
+import DialogContentContainer from "../../../components/DialogContentContainer/DialogContentContainer";
+import styles from "./dropdown-in-modals.module.scss";
+
+<Meta title="Technical Patterns/Dropdowns inside pop-overs" />
+
+<!--- Component documentation -->
+
+<ComponentNameDecorator withFoundationBackground>Display dropdowns inside pop-overs</ComponentNameDecorator>
+
+- [Overview](#overview)
+- [Up next](#up-next)
+- [Feedback](#feedback)
+
+## Problem description
```

Comment:
as suggested:
1. split this section to use cases (cut by overflow, hidden behind another content, ...)
2. have a single solution prop (`attachTo: 'current' | 'scrollParent' | CUSTOM_SELECTOR`)
3. consider simplifying example (box with black borders + overflow)

Reply:
as we discussed we keeping the content and not changing the API in this stage (will consider this later)

Reply:
(Keeping examples with vibe components because this are the common usecases which problematic to developers are with modal and dialog components)